rviz2/package.xml

```diff
@@ -1,3 +1,7 @@
+<?xml version="1.0"?>
+<?xml-model
+  href="http://download.ros.org/schema/package_format2.xsd"
```

Comment:
I've noticed that some packages are using package_format2.xsd while others are using package_format3. Are we generally trying to update to 3?

Reply:
Unless we want to use some of the new features of format 3, I don't think updating is necessary.

Reply:
Ok, just checking.